mne/io/meas_info.py

```diff
@@ -40,6 +40,7 @@
     eog=(FIFF.FIFFV_EOG_CH, FIFF.FIFFV_COIL_NONE, FIFF.FIFF_UNIT_V),
     ecg=(FIFF.FIFFV_ECG_CH, FIFF.FIFFV_COIL_NONE, FIFF.FIFF_UNIT_V),
     seeg=(FIFF.FIFFV_SEEG_CH, FIFF.FIFFV_COIL_NONE, FIFF.FIFF_UNIT_V),
+    bio=(FIFF.FIFFV_BIO_CH, FIFF.FIFFV_COIL_EEG_BIPOLAR, FIFF.FIFF_UNIT_V)
```

Comment:
> I wonder if it is safe to set it as EEG_BIPOLAR.
> 
Set it to coil none 

It's less magic
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
>